Solutions/ForescoutHostPropertyMonitor/Analytic Rules/ForeScout-DNSSniffEventMonitor.yaml

```diff
@@ -0,0 +1,30 @@
+id: d272e277-f285-4dbc-ae2d-7f65ba64a79e
+name: Forescout-DNS_Sniff_Event_Monitor
+description: 'This rule creates an incident when more than certain number of Dnsniff events are generated from a host'
+severity: Medium
+requiredDataConnectors: []
+queryFrequency: 5m
+queryPeriod: 5m
+triggerOperator: gt
+triggerThreshold: 0
+tactics: []
+relevantTechniques: []
+query:
+  ForescoutHostProperties_CL
+  | where HostProperties_DnsniffEvent_s <> ''
+  | where HostProperties_DnsniffEvent_s matches regex "DNS Query Type:.A;DNS Query/Response:.Query"
```

Comment:
If there is a regex match on `HostProperties_DnsniffEvent_s`, then why the check above for an empty string?

Reply:
Removed redundancy check on empty string.